desktop/desktop/lib/state/README.md

```diff
@@ -0,0 +1,6 @@
+State
```

Comment:
I think a decent strategy might be to continue with these suggestions in PR's based off of this branch (vs master)... That way we can keep PR's well scoped - avoiding a situation where reviewers are constantly reviewing a moving target, but at the same time we're not rushed in to merging to master in a state that's less than ideal.
This is what I've started doing with some of the other changes that I've been working on, like #12537.

My only worry with doing it this way is that work is still being done over at `wp-desktop` and making sure that this work is included in the merge is a fairly manual task... one that will become harder as the two become further out of sync. Luckily the rate of change there is much slower so this might not be a huge issue.